gpuflow/dataplane/gpu/cuda/cuda_lpm_factory.cu

```diff
@@ -160,7 +160,13 @@ int IPv6LPMFactory::AddLPMRule(uint8_t *ipv6_address, uint8_t depth, uint8_t nex
 
   }
 
-  SetupIPv6RuleEntry<<<1, end -start>>>(IPv6TBL24, start, next_hop, depth);
+  if (end < start){
+    SetupIPv6RuleEntry<<<1, 1>>>(IPv6TBL24, start, next_hop, depth);
+  } else if ((end - start) <= 1024) {
+    SetupIPv6RuleEntry<<<1, (end-start)>>>(IPv6TBL24, start, next_hop, depth); 
+  } else {
+    SetupIPv6RuleEntry<<<(end - start)/1024, 1024>>>(IPv6TBL24, start, next_hop, depth);
```

Comment:
It's better to name a variable `num_of_threads = 1024` above.

Reply:
ok, I'll add it.